src/components/AddBook.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+function AddBook() {
+  return (
+    <div>
+      <h2 className="text-xl text-[#888] font-bold uppercase tracking-tight mb-4">
+        Add new book
+      </h2>
+      <form className="flex items-center justify-between" action="">
+        <input
+          type="text"
+          name="bookTitle"
+          id="bookTitle"
+          placeholder="Book title"
+          className="bg-white border text-slate-400 px-4 py-2 w-[40rem] border-[#e8e8e8] outline-none text-sm"
+        />
+        <select
+          name="category"
+          id="category"
+          className="bg-white border text-slate-400 px-4 py-2 w-60 border-[#e8e8e8] text-sm "
+        >
+          <option value="" selected disabled>
+            Category
+          </option>
+          <option value="action">Action</option>
+          <option value="science fiction">Science Fiction</option>
+          <option value="economy">Economy</option>
+        </select>
```

Comment:
- [ ] @BahirHakimy as previously requested kindly change the category input to the author input, as I have highlighted below it's a project requirement.
![image](https://user-images.githubusercontent.com/61408860/230666093-60f9eb79-24b9-441c-8cf5-6b5bf9a07509.png)